mlflow/pipelines/steps/train.py

```diff
@@ -457,6 +491,35 @@ def render_schema(inputs, title):
         else:
             run_card_tab.add_markdown("MODEL_URI", f"**MLflow Model URI:** `{model_uri}`")
 
+        # Tab 8: Best Parameters
+        if best_estimator_params:
+            best_parameters_card_tab = card.add_tab(
+                "Best Parameters",
+                "{{ BEST_PARAMETERS }} ",
+            )
+            best_parameters_yaml = os.path.join(output_directory, "best_parameters.yaml")
+            if os.path.exists(best_parameters_yaml):
+                best_hardcoded_parameters = open(best_parameters_yaml).read()
+                best_parameters_card_tab.add_html(
+                    "BEST_PARAMETERS",
+                    f"<b>Best parameters:</b><br>"
+                    f"<pre>{best_hardcoded_parameters}</pre><br><br>",
+                )
+
+        # Tab 9: HP trials
+        if tuning_df is not None:
+            tuning_trials_card_tab = card.add_tab(
+                "Tuning Trials", "{{ SEARCH_SPACE }}" + "{{ TUNING_TABLE }}"
+            )
+            tuning_params = yaml.dump(self.step_config["tuning"]["parameters"])
+            tuning_trials_card_tab.add_html(
+                "SEARCH_SPACE",
+                f"<b>Tuning search space:</b> <br><pre>{tuning_params}</pre><br><br>",
+            )
+            tuning_trials_card_tab.add_html(
```

Comment:
Can we add some text before this table to indicate that these are the tuning results? "Tuning Results" seems good. Can we sort the results by the primary metric such that the first result is the best and also bold the first result? Looks like our sort order is incorrect (worst model first) in the screenshot you've provided in the description.